pronotepy/dataClasses.py

```diff
@@ -247,7 +247,7 @@ def __init__(self, client: _ClientBase, json_dict: dict) -> None:
     @property
     def grades(self) -> List["Grade"]:
         """Get grades from the period."""
-        json_data = {'periode': {'N': self.id, 'L': self.name}}
+        json_data = {'Periode': {'N': self.id, 'L': self.name}}
```

Comment:
This wasn't working at all? How did our tests pass?

Reply:
Because the request work but the server doesn't understand the request, so he send all the grade which are store in the database.

Reply:
Oooh... Alright then

Reply:
Oooh... Alright then